src/FormModel.php

```diff
@@ -328,11 +331,19 @@ private function generateAttributeLabel(string $name): string
 
     private function readProperty(string $attribute)
     {
-        $class = get_class($this);
+        $class = static::class;
         if (!property_exists($class, $attribute)) {
             throw new InvalidArgumentException("Undefined property: \"$class::$attribute\".");
         }
 
+        $getter = 'get' . $this->inflector->camelize($attribute);
+        if (is_callable([$this, $getter], true)) {
+            $method = new ReflectionMethod($class, 'get' . $this->inflector->camelize($attribute));
```

Comment:
```suggestion
            $method = new ReflectionMethod($class, $getter);
```

Reply:
The consequence of drafting and not refactoring correctly using IDE tools :)